src/components/CheckoutHeader/CheckoutHeader.stories.jsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import { CheckoutHeader } from "./CheckoutHeader";
+
+export default {
+    component: CheckoutHeader,
+    title: "Components/CheckoutHeader",
+};
+
+export const Default = () => {
+    return (
+        <div style={{ width: "375px" }}>
+            <CheckoutHeader
+                pizzaCounter="2"
+                totalPrice="(£26.00)"             
```

Comment:
the header should take a price in pence like other comps and format it itself inside